src/common/k8s-api/kube-object.store.ts

```diff
@@ -377,7 +377,7 @@ export abstract class KubeObjectStore<T extends KubeObject> extends ItemStore<T>
   // collect items from watch-api events to avoid UI blowing up with huge streams of data
   protected eventsBuffer = observable.array<IKubeWatchEvent<KubeJsonApiData>>([], { deep: false });
 
-  protected bindWatchEventsUpdater(delay = 1000) {
+  protected bindWatchEventsUpdater(delay = 4000) {
```

Comment:
Arbitrary delays are bad. Could you explain why it is needed?

Reply:
Increased delay because triggering `bindWatchEventsUpdater` every second results very costly table re-render process as well as bunch of Array manipulations (currently trying to optimise them in another branch).

Reply:
The down-level method already has `@action` decorator on it.

Reply:
True that should be good enough for grouping rerenders